openshift/openshift-ansible-contrib
# Add Support for Single Vm Deployment

#### What does this PR do?
This PR adds support for Single Virtual Machine Deployment of OCP.

#### How should this be manually tested?
Create a deployment using the template, verify that resource group is created, one vm is created,
and that openshift is installed, and a app can be launched.

#### Is there a relevant Issue open for this?
None

#### Who would you like to review this?
cc: @e-minguez  @cooktheryan  PTAL
